src/layout/Technologies/Technologies.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import  { Technology } from '../../components/Technology'
+
+class Technologies extends Component {
+  render() {
+    return (
+      <div className='technologies'>
+        <div className='technologies__title'>
+          <h1>Stuff we use to build stuff</h1>
+        </div>

```

Comment:
Since we are using (BEM)[http://getbem.com/introduction/] we don't actually need a `<h1>` tag, because you can style your element the way you want only by using the BEM method - Block, Element, Modifier, in this case you are applying correctly **Block** - `technologies` , **Element** - `title`, `techonologies__title`
So you only need to have 
```
<div className='technologies__title'>
   Stuff we use to build stuff
</div>
```

and change in SCSS file as well *

Reply:
😲